readme.org

```diff
@@ -121,9 +121,9 @@ more.
 
    The rules are more-or-less sorted by priority.
 
-   0. Don't bind anything to ~:~ nor ~<escape>~.
```

Comment:
Lets keep it at 0, I think it was intended (we're programmers afterall :)).

Reply:
Emacs automatically changes that to 1. It doesn't matter after all. It's a cosmetic thing.

Reply:
What does that mean? How does your emacs change a 0 to a 1?

Reply:
Emacs automatically changes the numbering of an ordered list when I edit the file. I don't know why it's doing that. Just let it go. I don't want to think too much about inconsequential cosmetic details.